.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,32 +1,26 @@
 ---
 name: Feature request
-about: Suggest an idea for this project
-title: "[Feature] - "
-labels: feature
+about: Suggest an idea for XPlat
+title: "[Feature] "
+labels: feature-request
 assignees: ''
 
 ---
 
-## What type of feature report is this?
-> Please check one or more that apply
+## Description
+<!-- Please describe the idea you'd like to see implemented -->
 
-- [ ] New feature
-- [ ] Improvement to existing feature
-- [ ] Other (include details)
+## How will this benefit you and others?
+<!-- Please describe what benefit the introduction of the feature will bring to your project -->
 
-## What is the behaviour that will be introduced?
-> Please describe the change you'd like to see introduced
+## What is the expected API coverage?
+<!-- Please provide a list of the events and methods expected in the covered API you'd expect to see implemented -->
```

Comment:
This comment asks to list event and methods but the sections provided include properties. 

Is this intentional?

Reply:
Added an update to this to include properties. Well spotted